src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -48,7 +50,9 @@
                        :icon-only? icon-name
                        :size       32
                        :accessible true
-                       :background (when (button-properties/backgrounds background) background))
+                       :background (if overlay-shown?
```

Comment:
perhaps put this logic into a simple helper function and leave a comment by that function explaining it's purpose there?

Reply:
We just check if the background is within a set of button types, Here's the definition of [backgrounds](https://github.com/status-im/status-mobile/blob/0531adb3ddd5a3c6e8a69c3b26f47f1105546350/src/quo2/components/buttons/button/properties.cljs#L4).

Reply:
I more so meant this because of the overlay part. It's somewhat of a design hack and so isolating to its own function and leaving the comment there hopefully makes the reason for this more apparent/easy to see

Reply:
I think it's better to have a comment explaining it than having a function for it.